.github/workflows/tests.yml

```diff
@@ -14,33 +14,22 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-
       - uses: ./.github/actions/setup-goversion
-
       - name: Build
         run: go build -v ./...
-
       - name: Lint
         uses: golangci/golangci-lint-action@971e284b6050e8a5849b72094c50ab08da042db8 # 6.1.1
         with:
           version: v1.56
-
       - name: Test
         run: go test -v ./...
-  check-for-doc-changes:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 #v4.2.2
-      - name: Install make
-        run: sudo apt-get update && sudo apt-get install -y make
-        shell: bash
-      - name: Regenerate Helm Docs
```

Comment:
Removing the helm test because it was duplicative of the test in `helm.yml` workflow.